server.js

```diff
@@ -17,6 +18,23 @@ app.get('/api/foods', FoodsController.index);
 
 app.post('/api/foods', FoodsController.create);
 
+app.patch('/api/foods/:id', FoodsController.update);
+
+// app.patch('/api/foods/:id', (req, res) =>{
```

Comment:
We can remove all of this commented code once we go over this, or we find another solution